### DISCUSSION TITLE
Tired of seeing your data in rails console having so many lines? ...

### DISCUSSION THREAD

**Discussion by annagabriel-hash** [avionschool/batch8-activities#28] [Show and tell]
### Want to see your data 
**From this...**
![image](https://user-images.githubusercontent.com/79251819/130244071-7be1cca8-f4f9-46cb-bd3b-0a5556bc5b50.png)

**To this?**
![image](https://user-images.githubusercontent.com/79251819/130243890-150f6865-0cb8-449e-ab13-df0db9c26e7a.png)

### Introducting **Hirb gem** 😄 

Installation
1. Add Hirb gem to dev and test environment
```
gem 'hirb'
```
2. Run `bundle install`
3. Open `rails console`
4. Enter `Hirb.enable`
5. Display your data
![image](https://user-images.githubusercontent.com/79251819/130244452-c48adbe8-7f37-4019-bef4-0d1041bdb928.png)

PS. Other gem alternatives
`table_print`
`awesome_print`
Ref: [Rails console: six ways to get pretty output](https://aneta-bielska.github.io/blog/rails-c-5-ways-to-get-pretty-output.html)

**Comment by lea-tinoso** [comment]
Whoa!!!! 👏👏👏👏. This reminds me of what @theIanMilan  shared during our front-end class, using `console.table` instead of `console.log` especially useful when viewing 2D arrays.
